packages/perseus/src/widgets/radio/prompt-utils.ts

```diff
@@ -0,0 +1,27 @@
+import {WidgetType, type RadioPromptJSON} from "../../prompt-types";
+
+import type {RenderProps} from "./radio-component";
+import type {PerseusRadioUserInput} from "../../validation.types";
+
+export const getPromptJSON = (
+    renderProps: RenderProps,
+    userInput: PerseusRadioUserInput,
+): RadioPromptJSON => {
+    const {choices} = renderProps;
```

Comment:
Unless you want me to, I'm not really paying attention to what information you're getting (which I feel like is a TUT concern), but how you're getting it (which is a LEMS concern). lmk if you want more feedback here, but also I think it'll help to have [LEMS-2471](https://khanacademy.atlassian.net/browse/LEMS-2471) done so you have a clearer view of what you might need.

Reply:
I'm going to collaborate with one of our prompt engineers to check that we're getting the correct information. I'll be keeping an eye on that ticket. Thanks!